### PR TITLE
Adds link to blog post authors name

### PR DESCRIPTION
**Pull request by daniellemaxwell**
Fixes SC-19502

- Creates authors page for each blog post author
- Displays author bio and blog posts on one page
- Adds link to author's name on blog posts
- Replaces link on team page to direct users to the authors page

Acceptance Criteria:
https://www.awesomescreenshot.com/video/21309206?key=ed09b16251097431d58482ff9d816c66

Note: In the video above, you'll notice that the pages for authors who do not have an `authors` file for them in `content` only displays their blog posts.

### CODE REVIEW

File: archetypes/blog.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ date: "{{ .Date }}"
 draft: false
 image: img/blog/
 photo_credit: "Add Photo Credits Here"
-author: Your Name
+authors: ['Your Name', 'Co-Authors Name']
```
Comment (daniellemaxwell) [inline archetypes/blog.md:8]: Hugo requires that taxonomies are plural, so each blog post required that `author` be replaced with `authors`. The author's name also has to be enclosed with brackets, even for single names. I decided to go ahead and include an update to the archetype file.

File: config.toml [hunk 2]
```diff
@@ -159,3 +159,7 @@ blog = ["HTML", "RSS"]
 baseName = "humans"
 isPlainText = true
 mediaType = "text/plain"
+
+[taxonomies]
+  tag = 'tags'
```
Comment (daniellemaxwell) [inline config.toml:164]: Although `tags` is a default Hugo taxonomy, it had to be added here to continue to work now that the custom `authors` taxonomy has been added.

File: content/en/authors/aatmaj-janardanan/_index.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+---
```
Comment (daniellemaxwell) [inline content/en/authors/aatmaj-janardanan/_index.md:1]: To have an authors page that displays blog posts, `bio` will be replaced with `authors`. We also have to follow this file structure (`content/en/authors/author-name/_index.md`) when adding a new author/bio to the site. I'll create a separate story to remove the `bio` files.

File: layouts/blog/single.html [hunk 4]
```diff
@@ -26,7 +26,9 @@ <h3 class="font-bold text-xl sm:text-2xl lg:text-3xl text-center" data-blog-titl
       </h3>
       <div class="flex flex-wrap justify-center items-center my-4">
         <img src="{{ .Params.profile }}" alt="" class="border-4 border-white rounded-full h-11 drop-shadow-lg">
-          <span class="ml-3 blog-date">{{ .Params.author }} | {{ dateFormat "Monday, Jan 2, 2006" .Date }} |&nbsp; </span>
+        {{ range .Params.authors }}
+          <span class="ml-3 blog-date"><a href="/authors/{{ . | urlize }}">{{ . }}</a>  {{ end }} | {{ dateFormat "Monday, Jan 2, 2006" .Date }} |&nbsp; </span>
```
Comment (daniellemaxwell) [inline layouts/blog/single.html:30]: The names are not displayed on one line anymore. I'll create a follow on story to fix that.

File: layouts/authors/list.html [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+
+<div class="max-w-7xl mx-auto px-4 my-auto relative">
+    <div class="flex basis-3 items-start py-4 md:p-16 flex-col md:flex-row">
+        {{ if .Params.name }}
+        <div class="w-full md:w-1/2">
+            <div class="">
+                {{ if .Params.profile }}
+                <img src="{{ .Params.profile | absURL}}" alt="" class='rounded-t-xl covatar w-full object-cover' />
+                {{ else }}
+                <img src="/img/butterfly.png" alt=""  />
```
Comment (daniellemaxwell) [inline layouts/authors/list.html:24]: I've added this photo as more of a placeholder for contributing authors, if we don't have a photo for them. We can replace this with something else.

File: layouts/authors/terms.html [hunk 6]
```diff
@@ -0,0 +1,6 @@
+<h1>Authors</h1>
```
Comment (daniellemaxwell) [inline layouts/authors/terms.html:1]: This provides style for an `/authors` page. It's basic now, so I can create a follow on story to design and style it.

File: content/en/authors/aatmaj-janardanan/_index.md (hunk 3 above)
Comment (rebeccabilbro) [reply content/en/authors/aatmaj-janardanan/_index.md:1]: Could we also do a redirect that makes sure if someone goes to one of the old bio pages, (eg `https://rotational.io/bio/rebecca-bilbro/`) they get rerouted to whatever the new url will be so they don't see a broken link?

File: layouts/authors/list.html (hunk 5 above)
Comment (rebeccabilbro) [reply layouts/authors/list.html:24]: That works for me!

**Review by rebeccabilbro — APPROVED** [review]
Approved @daniellemaxwell ! I have one question that relates more to the change from https://rotational.io/bio/author-name/ to (i think) https://rotational.io/authors/author-name/, but I think the new pages look great, and this is precisely the functionality I was hoping for!

File: content/en/authors/aatmaj-janardanan/_index.md (hunk 3 above)
Comment (daniellemaxwell) [reply content/en/authors/aatmaj-janardanan/_index.md:1]: Sure, I'll include that as part of the acceptance criteria when deleting the bio files.

**Pull request merged**